tests/resume.test.ts

```diff
@@ -217,6 +216,69 @@ describe('patch resume', () => {
     const resumeController = ControllerFactory.resume(conn);
     await expect(
       resumeController.patchResume(params, request, anotherMember),
-    ).rejects.toThrowError(ForbiddenError);
+    ).rejects.toThrowErrorMatchingInlineSnapshot(
+      '"Cannot update a resume of another user"',
+    );
```

Comment:
What does this method do? What is an inline snapshot? Is this a better alternative to just using `toThrow('Cannot update a resume of another user')`?

Reply:
https://jestjs.io/docs/snapshot-testing#inline-snapshots
https://jestjs.io/docs/expect#tothrowerrormatchinginlinesnapshotinlinesnapshot

It doesn't seem to me that inline snapshots are too relevant to what we want to assert, is there a use case that is useful for us here?

Reply:
I did not realize toThrow works with strings as well. I was certain vscode was shouting at me when I tried it initially. Inline snapshot, however, works here as it throws when I have a different string.

Reply:
I still think we should stray away from using this method because it is not relevant to the testing we are trying to do here, even though the method may provide the same expected results we are looking for. It looks like "inline snapshots" are a concept related to [Snapshot Testing](https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest), which is meant specifically for UI testing, as explained in the docs.

Including this method here only causes more confusion because in every other test we assert exceptions with `toThrow`, which provides the expected behavior while also semantically making sense - we expect an error to be thrown with this specific error message, and nothing more.